src/core/book-keeper.js

```diff
@@ -1,33 +1,34 @@
 'use strict';
 
 var events = require('./event/event');
+var interval = require('../../config/default');
```

Comment:
Bit nitpicky, but name this 'config' because 1) it's the convention set by the rest of the project and 2) default.js contains more than just an interval